core/src/main/resources/error/error-classes.json

```diff
@@ -14,13 +18,25 @@
   "CANNOT_EVALUATE_EXPRESSION" : {
     "message" : [ "Cannot evaluate expression: %s" ]
   },
+  "CANNOT_FETCH_TABLES_OF_DATABASE" : {
+    "message" : [ "Unable to fetch tables of db %s" ],
+    "sqlState" : "42000"
```

Comment:
This seems to be a catch-all type exception. I wouldn't group this as a syntax/semantic error (42000); maybe we can leave the SQLSTATE field empty.